dreem/io/config.py

```
@@ -197,7 +199,7 @@
         mode: str,
         label_files: list[str] | None = None,
         vid_files: list[str | list[str]] = None,
-    ) -> "SleapDataset" | "MicroscopyDataset" | "CellTrackingDataset":
+    ) -> "SleapDataset" | "MicroscopyDataset" | "CellTrackingDataset" | None:
```

Ruff

Undefined name `SleapDataset`

(F821)